client/landing/gutenboarding/stores/mapped-types.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Mapped types
+ *
+ * This module should only contain mapped types, operations useful in the type system
+ * that do not produce any runtime code.
+ *
+ * Mapped types can be thought of as functions in the type system, they accept some type
+ * argument and transform it to another type.
+ *
+ * @see https://www.typescriptlang.org/docs/handbook/advanced-types.html#mapped-types
+ */
+
+/**
+ * Maps a "raw" selector object to the selectors available when registered on the wp-data store.
```

Comment:
Maybe
```suggestion
 * Maps a "raw" selector object to the selectors available when registered on the wordpress/data store.
```

since we're also referring to it by its full name [further down](https://github.com/Automattic/wp-calypso/pull/37511/files#diff-da39c5858be2c4d6264f652781603c6aR36) (and it might be somewhat clearer).